### PR TITLE
Added information about depot_tools and breakpad issues for VS2015 build instructions

### PR DESCRIPTION
**Pull request by OrBin**

### CODE REVIEW

**Comment by stek29** [comment]
Could you squash your commits?

**Comment by OrBin** [comment]
Squashed the commits.

File: docs/building-msvc.md [hunk 1]
```diff
@@ -251,11 +252,22 @@ If you get errors like
 
 run `git reset --hard HEAD` and execute `gclient` again
 
+If you get errors like
+
+    [P15084 09:15:28.171 fs.go:341 W] fs: failed to rename() - Access is denied.
+    [P15084 09:15:28.194 client.go:1378 E] Failed to install infra/tools/luci/led/windows-amd64:cc7454629aa4b4a90a5de014f4089cc680d15c4b - Access is denied. 
+    Error: failed to update packages, see the log.
+
+run `set PATH=D:\TBuild\Libraries\depot_tools;%PATH%` and execute `gclient` again
```
Comment (auchri) [inline docs/building-msvc.md]: missing point

File: docs/building-msvc.md [hunk 2]
```diff
@@ -251,11 +252,22 @@ If you get errors like
 
 run `git reset --hard HEAD` and execute `gclient` again
```
Comment (auchri) [inline docs/building-msvc.md]: Please add an point here too, even it is not related to your PR.

**Comment by auchri** [comment]
@OrBin I'm closing this, since there are outstanding changes and there is a merge conflict now.

Please correct the requested changes, remove the merge conflict and open a new PR.

**Comment by john-preston** [comment]
I don't think it is required, because now depot_tools are not used.

**Pull request closed without merging**